charts/rd-professional-api/values.preview.template.yaml

```diff
@@ -17,5 +18,4 @@ java:
       secrets:
         - LD-SDK-KEY
         - AppInsightsInstrumentationKey
-
-
+        - s2s-secret
```

Comment:
Please use professional-api-s2s-secret. s2s-secret will be deleted in the future.

Reply:
Change completed here: https://github.com/hmcts/rd-professional-api/pull/954/commits/db195119a1200e625b50e0a477708b50f07c17ac